test/sentiment-analysis.test.js

```diff
@@ -0,0 +1,39 @@
+
+const sentimentAnalysis = require('../src/sentiment-analysis');
+
+const expected = {
+  score: 1,
+  comparative: 0.1111111111111111,
+  calculation: [{ allergic: -2 }, { love: 3 }],
+  tokens: [
+    'i',
+    'love',
+    'cats',
+    'but',
+    'i',
+    'am',
+    'allergic',
+    'to',
+    'them',
+  ],
+  words: [
+    'allergic',
+    'love',
+  ],
+  positive: [
+    'love',
+  ],
+  negative: [
+    'allergic',
+  ],
+};
+
+test('test if sample text does not return the correct object', async () => {
+  const data = await sentimentAnalysis.run('I like dogs, but my girlfriend is afraid of them');
+  expect(data).toEqual(expect.not.objectContaining(expected));
```

Comment:
Why do you not just compare `data` to `expected` with a deep equals here?  What does `.toEqual(expect.not.objectContaining(expected))` mean?  Maybe you're right, but this seems odd.

Reply:
@humphd From what I have read, `.toEqual` should be used for objects. Regards to `expect.not.objectContaining(expected)` , I am using this to check if the contents(properties) of the object do not match what we are expecting. This was suggested by @jatinAroraGit and after reading the docs I agreed to his suggestion.

Reply:
Interesting, OK, sounds fine.  Thanks for educating me.